opa/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,21 @@
+name: opa
+files:
+- name: opa.yaml
```

Comment:
The `auto_conf.yaml` should be specified in this `spec.yaml` as well

Reply:
👍